rfc034-ruby-193-eol.md

```diff
@@ -5,9 +5,10 @@ Status: Accepted
 Type: Standards Track
 ---
 
-# Ruby 1.9.3 EOL
+# Ruby Version Deprecation Policy
 
-Chef 12.0 will require Ruby 2.0 or greater.
+Establishes a rolling version policy for Chef and ChefDK and the
+assocated ecosystem (berkhelf, test-kitchen, etC).
```

Comment:
"berkhelf" and "assocated" as well

i think i typed this out when i was high on too much coffee and gem pinning rage...